Task_1.2/BWT.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Task_1._2
+{
+    public class BWT
+    {
+        private static string getUniqueSymbols(string s)
+        {
+            var uniqueSymbols = new SortedSet<Char>();
+            foreach (var ch in s)
+                uniqueSymbols.Add(ch);
+            var new_string = "";
+            foreach (var ch in uniqueSymbols)
+            {
+                new_string += ch;
+            }
+
+            return new_string;
+        }
+
+        private static int getLessSymbolsCount(string s, char symbol)
+        {
+            var cnt = 0;
+            foreach (var ch in s)
+            {
+                if (ch < symbol)
+                    cnt++;
+            }
+
+            return cnt;
+        }
+
+        private static string getSortedString(string s)
+        {
+            var list = s.ToList();
+            list.Sort();
+            var sortedString = "";
+            foreach (var ch in list)
+                sortedString += ch;
+            return sortedString;
+        }
+
+        private static int countSymbolInSortedString(string sortedString, char symbol)
+        {
+            var count = 0;
+            var startIndex = sortedString.IndexOf(symbol);
+            for (var i = startIndex; i < sortedString.Length; ++i)
+            {
+                if (sortedString[i] == symbol)
+                    count++;
+                else
+                    break;
+            }
+
+            return count;
+        }
+
+        private static int countSymbolInString(string s, char symbol)
+        {
+            var cnt = 0;
+            foreach (var ch in s)
+            {
+                if (ch == symbol)
+                    cnt++;
+            }
+
+            return cnt;
+        }
+
+        // прямой BWT
+        public static KeyValuePair<string, int> StraightBWT(String s)
```

Comment:
```suggestion
        public static (string transformedText, int position) StraightBWT(String s)
```
Для описания типа пары стоит использовать кортежи, а не служебный тип словарей.
(-0.2 балла)